guides/v2.3/graphql/reference/downloadable-product.md

```diff
@@ -90,3 +106,134 @@ The following query returns information about downloadable product `240-LV04`, w
    }
 }
 ```
+**Response**
+
+```json
+{
+  "data": {
+    "products": {
+      "items": [
+        {
+          "id": 47,
+          "name": "Beginner's Yoga",
+          "sku": "240-LV04",
+          "type_id": "downloadable",
+          "price": {
+            "regularPrice": {
+              "amount": {
+                "value": 6,
+                "currency": "USD"
+              }
+            }
+          },
+          "links_title": "Downloads",
+          "links_purchased_separately": 0,
+          "downloadable_product_links": [
+            {
+              "id": 1,
+              "sample_url": null,
+              "sample_type": null,
+              "is_shareable": false,
+              "number_of_downloads": 0,
+              "sort_order": 1,
+              "title": "Beginner's Yoga",
+              "link_type": "FILE",
+              "price": 6
+            }
+          ],
+          "downloadable_product_samples": [
+            {
+              "title": "Trailer #1",
+              "sort_order": 1,
+              "sample_type": "FILE",
+              "sample_file": "/l/u/luma_background_-_model_against_fence_4_sec_.mp4"
+            },
+            {
+              "title": "Trailer #2",
+              "sort_order": 1,
+              "sample_type": "FILE",
+              "sample_file": "/l/u/luma_background_-_model_against_fence_4_sec_.mp4"
+            },
+            {
+              "title": "Trailer #3",
+              "sort_order": 1,
+              "sample_type": "FILE",
+              "sample_file": "/l/u/luma_background_-_model_against_fence_4_sec_.mp4"
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+
+## Customer Downloadable Product {#customerDownloadProduct}
```

Comment:
Since this query is defined in this module, move it so that it precedes the `products` query. Since ConfigurableProduct module only defined a mutation, it should be after the query.

Reply:
moved and created new section:
## Downloadable product query
The `DownloadableProduct` query returns downloadable product information when you perform a `products` search.